Signal/src/environment/OWSPreferences.m

```diff
@@ -270,8 +270,8 @@ - (nullable NSString *)getVoipToken
 - (void)unsetRecordedAPNSTokens
 {
     DDLogWarn(@"%@ Forgetting recorded APNS tokens", self.tag);
-    [self setValue:nil forKey:OWSPreferencesKeyLastRecordedPushToken];
-    [self setValue:nil forKey:OWSPreferencesKeyLastRecordedVoipToken];
+    [self setValueForKey:OWSPreferencesKeyLastRecordedPushToken toValue:nil];
```

Comment:
This is unrelated to the bug, but a small fix to a debug action that I came upon while testing this.

Reply:
Nice.